src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,88 @@
+package duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+
+/**
+ * The Storage class deals with loading tasks from the data file and
+ * saving tasks in the file.
+ *
+ * @author Edric Yeo
+ */
+public class Storage {
+
+    private String filePath;
```

Comment:
You could add a JavaDoc for class member. You haven't done that for a few other class members in other classes too.